tests/run-make/print-request-help-stable-unstable/rmake.rs

```diff
@@ -0,0 +1,31 @@
+//! For issue-138698
+use run_make_support::{diff, rustc, similar};
+
+fn main() {
+    // Failed. stable rustc print invalid print request help
+    let stable_invalid_print_request_help = rustc()
+        .env("RUSTC_BOOTSTRAP", "-1")
+        .cfg("force_stable")
+        .print("xxx")
+        .run_fail()
+        .stderr_utf8();
```

Comment:
Suggestion: here, cherry-pick one print request that we certainly don't expect to be stable any time soon (or maybe even ever), e.g. `all-target-specs-json`

- In the *stable* help, check that the stderr does not include a mention of `all-target-specs-json`.
- In the *unstable* help, assert that `all-target-specs-json` is present.

This is to make the snapshots harder to accidentally bless away.

(And leave a comment about cherry-picking `all-target-specs-json` as an example)